src/common/utils/time-utils.ts

```diff
@@ -1,31 +1,41 @@
 import { DateTime } from 'luxon';
-import moment from 'moment';
 
 export enum TimeFormat {
   YYYY_MM_DD = 'yyyy-MM-dd', // 2020-03-01
   MM_DD_YYYY = 'MM/dd/yyyy', // 03/01/2020
   MMM_DD_YYYY = 'MMM dd, yyyy', // Mar 01, 2020
   MMM_D_YYYY = 'MMM d, yyyy', // Mar 1, 2020
   MMMM_D_YYYY = 'MMMM d, yyyy', // March 1, 2020
+  MMM_YY = 'MMM yy', // Mar 20.
   MMMM_D = 'MMMM d', // March 1
+  MMM = 'MMM', // Mar
 }
 
 export enum TimeUnit {
-  HOURS = 'hours',
-  DAYS = 'days',
-  WEEKS = 'weeks',
-  MONTHS = 'months',
+  HOURS,
+  DAYS,
+  WEEKS,
+  MONTHS,
```

Comment:
It's better to add a string values to enums to make it easier to debug (unless the enum represents numeric amounts)

```ts
export enum TimeUnit {
  HOURS = 'hours',
  DAYS = 'days',
  // ...
}
```

if we do that, we can use their string value in other places, for example

```ts
function subtractTime(date: Date, amount: number, unit: TimeUnit): Date {
  // If unit = TimeUnit.HOURS, then it becomes "hours" when
  // building and then we get amountOptions = { "hours": amount }; 
  const amountOption = { [unit]: amount };
  return DateTime.fromJSDate(date).minus(amountOption).toJSDate();
}
```

we can do the same for the `addTime` function